spec/display_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'display'
+
+describe Display do
+  let(:output) { StringIO.new }
+  let(:display) { Display.new(output) }
+
+  it "displays grid of 1 by 1 with letter A" do
+    display.display_grid(1)
+    expect(output.string).to include(" A ", ".")
+  end
+
+  it "Asks the player for a number coordinate" do
+    display.ask_for_number
+    expect(output.string).to eq("Time to make a move. Please enter a number from 1 - 10\n")
```

Comment:
Maybe you could use `.to include("Your message")` to make these kinds of tests less fragile. If the wording of your message changes in the future then the test would break. You could check that it only includes the most important parts of the message that are less likely to change e.g. "enter a number from 1 - 10".